lib/permutations.rb

```diff
@@ -1,4 +1,8 @@
-
 def permutations?(string1, string2)
+  h = Hash.new { |h, k| h[k] = 0 }
   
+  string1.each_char { |c| h[c] += 1 }
+  string2.each_char { |c| h[c] -= 1 }
+  
+  h.values.all?(&:zero?)
```

Comment:
Nice clean solution. My only piece of critical feedback is the same as for the palindrome method for naming variables/collections.